config/database.yml

```diff
@@ -0,0 +1,39 @@
+# Database access settings
```

Comment:
Those files should be added als *.dist, e.g. database.yml.dist (see [this](https://stackoverflow.com/questions/16843080/what-does-dist-used-as-an-extension-of-some-source-code-file-mean) for reference)

But don't put to much effort in stuff like this. We can take care of it before merging your branch.